.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 4.7.1
+current_version = 4.7.2
```

Comment:
Are you sure that this should be found in this pull request?

Reply:
Probably not.  Just take the first commit.  I did it for myself because I needed to reference the new version, and others who use my repo until it's merged in will need the version bump too...